packages/flutter/lib/src/material/slider.dart

```diff
@@ -124,6 +125,8 @@ class Slider extends StatefulWidget {
     this.activeColor,
     this.inactiveColor,
     this.semanticFormatterCallback,
+    this.focusNode,
```

Comment:
Are you also planning on adding the focusColor and hoverColor attributes that are common on other components? It might not be appropriate here, but seems like it could be.

Reply:
Not immediately, the design for the hover and focus was to basically just use the overlay